Makefile

```diff
@@ -25,7 +33,29 @@ ifdef USE_GIT
   endif
 endif
 
-CXXFLAGS:=$(CXXFLAGS) -I$(SNAPPY_SRC_DIR)
+ifdef ENABLE_BITSHUFFLE
```

Comment:
Yeah, we need this to turn sse/avx modes on/off. The mode is off by default, that is, the universal one is used for native libraries included in `snappy-java`.